ramalama/shortnames.py

```diff
@@ -0,0 +1,32 @@
+import configparser
+import os
+
+
+class Shortnames:
+    """Model super class"""
+
+    shortnames = {}
+
+    def __init__(self):
+        config = configparser.ConfigParser()
+        sn = {}
+        file_paths = [
+            "/usr/share/ramalama/shortnames.conf",  # for development
```

Comment:
This first directory was:

`usr/share/ramalama/shortnames.conf`

it was intended for when one calls ramalama.py in the git repo for testing. Not required, but useful.

Now that it has a slash, it's just a duplicate entry in the list.